stdtx/src/amino/msg/builder.rs

```diff
@@ -35,17 +33,14 @@ impl<'a> Builder<'a> {
     /// Create a new message builder for the given schema and message type
     pub fn new(
         schema: &'a Schema,
-        type_name: impl TryInto<TypeName, Error = Error>,
-    ) -> Result<Self, Error> {
+        type_name: impl TryInto<TypeName, Error = eyre::Report>,
```

Comment:
ooo we should serialize to JSON and send report to datadog. Time to revive https://github.com/iqlusioninc/crates/pull/492